### PR TITLE
Update: OpenWhisperSystems.Signal.Beta version 5.42.0-beta.1 [FP]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Signal Beta | Signal Beta 5.42.0-beta.1    |
| Version     | 5.42.0-beta.1     | 5.42.0-beta.1 |
| Publisher   | Signal Messenger, LLC   | Signal Messenger, LLC      |
| ProductCode |           | b39ac3a0-cc73-5ec2-ba18-5ab3c4de1ca1    |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [1373](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/2261672289>)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Alpha Version

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Whats up with the title?

File: manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider
+PackageVersion: 1.4.5.2031
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.2031
```

File: manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider
+PackageVersion: 1.4.5.2031
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.2031
```

File: manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider
+PackageVersion: 1.4.5.2031
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/1.4.5.2031/CiderCollective.Cider.yaml:5]:
```suggestion
PackageVersion: 1.4.5-alpha.2031
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**